### PR TITLE
new version for single PSQL instances

### PR DESCRIPTION
**Pull request by mfuhrmann**

### CODE REVIEW

**Comment by mfuhrmann** [comment]
@Svedrin Can you please merge this one?

File: v2/pgdeploy [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                return self.deploy(remote_host)
+            except pkg_resources.RequirementParseError:
+                sleep(1)
+                no_tries += 1
+        raise DeployError("exceeded number of retries for RequirementParseError workaround")
+
+    def deploy(self, remote_host):
+        cluster_config = "/etc/pgbouncer/databases/%s.ini" % self.cluster_name
+        pgpass = os.path.expanduser("~/.pgpass")
+        print remote_host["ip_addr"]
```
Comment (Svedrin) [inline v2/pgdeploy]: @mfuhrmann let's get rid of this print :)

**Review by Svedrin — CHANGES_REQUESTED** [review]
@mfuhrmann one nit, apart from that lgtm.

**Comment by mfuhrmann** [comment]
@Svedrin *ping*

**Review by Svedrin — APPROVED** [review]

**Pull request merged**